src/gar.lua

```diff
@@ -156,6 +156,13 @@ function gar.increaseOperatorStake(from, qty)
 end
 
 -- Utility function to calculate withdrawal details and handle balance adjustments
+---@param qty number # The amount of stake to withdraw
```

Comment:
Nit: some params specify that the amounts are in mIO, others don't. Perhaps we need something in the readme that specifies that all currency amounts used throughout the codebase are in mIO unless otherwise specified (and perhaps even that final qualifier need not apply since it's all mIO all the time)...

Reply:
we should never be concerned with IO in the contract, but agree these parameter names (and generally around the code base) can be more accurate. as we continue to update docs, we can standardize.

tweaked these here - [8f13aa5](https://github.com/ar-io/ar-io-network-process/pull/161/commits/8f13aa515d4798eeb8fe2920e20814ba6df7b6d9)